README.md

```diff
@@ -61,6 +61,19 @@ Feel free to ping [@mbland](https://github.com/mbland) with any questions you
 may have, especially if the current documentation should've addressed your
 needs, but didn't.
 
+### Contributing as an 18F Team Member
+
+1. Clone the repo, including the submodule for private data ( `git clone --recursive git@github.com:18F/hub.git` )
```

Comment:
Since this is the only step that's different, I'd just make this a sub-item in the earlier list. (And I didn't know about `git clone --recursive`. Nice!)